rest-api-spec/src/main/resources/rest-api-spec/api/indices.rollover.json

```diff
@@ -63,6 +63,10 @@
         "type":"boolean",
         "default":"false",
         "description":"If set to true, the rollover action will only mark a data stream to signal that it needs to be rolled over at the next write. Only allowed on data streams."
+      },
+      "failure_store":{
+        "type":"boolean",
+        "description":"If set to true, the rollover action will be applied on the failure store of the data stream."
```

Comment:
I completely missed this during the review, but we cannot add this query parameter here because it will expose it to our clients. 

Consequently, we cannot have yaml tests testing this feature until the feature flag is removed. :(